changelogs/fragments/485-fix-broken-ansible-build-data-link.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - "Fixed broken ansible-build-data repository link in ansible package readme (https://github.com/ansible-community/antsibull/pull/485)."
```

This should use present tense:

```suggestion
  - "Fix broken ansible-build-data repository link in ansible package README (https://github.com/ansible-community/antsibull/pull/485)."
```

---

Thanks, I'll keep that in mind!